dhis-2/dhis-api/src/main/java/org/hisp/dhis/fileresource/FileResource.java

```diff
@@ -79,6 +79,8 @@
      */
     private FileResourceDomain domain;
 
+    private boolean hasMultipleStorageFiles;
```

Comment:
What is the implication of having multiple files? Would be nice with some javadoc on this property to explain the intent.